<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>656</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTreeWidget" name="treeWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>131</width>
      <height>281</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>族谱管理系统</string>
     </property>
    </column>
    <item>
     <property name="text">
      <string>信息管理</string>
     </property>
     <item>
      <property name="text">
       <string>添加</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>修改</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>删除</string>
      </property>
     </item>
    </item>
    <item>
     <property name="text">
      <string>查询</string>
     </property>
     <item>
      <property name="text">
       <string>人员查询</string>
      </property>
     </item>
    </item>
    <item>
     <property name="text">
      <string>保存</string>
     </property>
     <item>
      <property name="text">
       <string>保存文件</string>
      </property>
     </item>
    </item>
    <item>
     <property name="text">
      <string>加载</string>
     </property>
     <item>
      <property name="text">
       <string>加载文件</string>
      </property>
     </item>
    </item>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>40</y>
      <width>431</width>
      <height>231</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;此系统为家族查询系统，系统基于树建立，实现了树的查询，删除，修改，树的遍历等操作。在查询系统上实现了模糊查询，最后会将读入的文件，在重开系统时可以通过加载文件实现复原。&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>280</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>预览</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>280</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>656</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>家族信息管理</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_2"/>
    <addaction name="action_3"/>
    <addaction name="action_7"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>查询</string>
    </property>
    <addaction name="action_4"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>保存</string>
    </property>
    <addaction name="action_5"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>加载</string>
    </property>
    <addaction name="action_6"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action">
   <property name="text">
    <string>成员添加</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>成员修改</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>成员删除</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>人员查询</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>保存入文件</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>加载文件</string>
   </property>
  </action>
  <action name="action_7">
   <property name="text">
    <string>创建族谱</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
